bin/build.sh

```diff
@@ -27,17 +35,19 @@ find 'edudoc' -name '*.md' | while read file; do
 
   pdf_name="${file%.md}.pdf"
   html_name="${file%.md}.html"
-
   header_html="${file%.md}-header.html"
-
+  edudoc_stylesheet="../../assets/edudoc-style.css"
```

Comment:
see above regarding realpath. Also, why the heck does this have two `..` but above it has three?!

Reply:
Because in one case it's two levels deep and in the other it's three